src/test/java/dk/aau/p4/abaaja/InterpreterTests/InterpreterTests.java

```diff
@@ -1057,7 +1057,6 @@ public void structAndListUse_setsValue(String code, boolean value) {
         IGameBridge bridgeMock = Mockito.mock(IGameBridge.class);
 
         concreteNode.accept(new Interpreter(problemCollection, symbolTable, bridgeMock));
-
         for (Problem problem : problemCollection.getProblems()) {
```

Comment:
```suggestion

        for (Problem problem : problemCollection.getProblems()) {
```
I am guessing this is just bing mistake

Reply:
Yup